src/Umbraco.Cms.Integrations.Search.Algolia/Services/ExtendedAlgoliaSearchPropertyIndexValueFactory.cs

```diff
@@ -0,0 +1,34 @@
+﻿
+using System.Text.Json;
```

Comment:
I think this class (and it's commented out registration) should probably be in the docs - for now, in the readme - rather than in the code.